### PR TITLE
 add i18n

### PR DESCRIPTION
**Pull request by AndriiBuhalo**
image

image

### CODE REVIEW

File: app/admin/products.rb [hunk 1]
```diff
@@ -59,7 +59,7 @@
       end
     end
 
-    panel ('Повязані продукти' + link_to('Додати продукт', add_related_products_admin_product_path(product), class: 'right panel_header_action')).html_safe do
+    panel "Повязані продукти#{link_to('Додати продукт', add_related_products_admin_product_path(product), class: 'right panel_header_action')}".html_safe do
```
Comment (Shabaldas) [inline app/admin/products.rb]: move it also in i18n.

File: app/views/shared/_footer.html.erb [hunk 2]
```diff
@@ -2,29 +2,31 @@
...
-				<p>вул. Тисменицька, 248</p>
-				<p>Івано-Франківськ</p>
+				<h4 class="text-zinc-400 uppercase"><%= t('global.footer.address')%></h4>
+				<p><%= t('global.footer.street') %></p>
+				<p><%= t('global.footer.city') %></p>
 			</div>
 		</div>
 		<div class="flex justify-between items-center text-zinc-400 border-t border-gray-500 pt-6">
-			<p>©2021 Всі права захищені</p>
+      <%= "© #{Time.new.year}" + t('global.footer.law') %>
```
Comment (Shabaldas) [inline app/views/shared/_footer.html.erb]: replace 
```Time.new.year```
to
```Time.zone.now.year```

File: app/admin/products.rb (hunk 1 above)
Comment (AndriiBuhalo) [reply app/admin/products.rb]: fix

File: app/views/shared/_footer.html.erb (hunk 2 above)
Comment (AndriiBuhalo) [reply app/views/shared/_footer.html.erb]: fix

**Pull request merged**